### PR TITLE
add fleet networking feature example

### PR DESCRIPTION
**Pull request by zhiying-lin**

### CODE REVIEW

File: examples/fleet/helloworld/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Example: Place HelloWorld application to  on Fleet
+# Example: Place HelloWorld application to on Fleet
```
Comment (circy9) [inline examples/fleet/helloworld/README.md:1]: Place HelloWorld application to multiple clusters in a fleet

File: examples/fleet/kuard/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# Example: Creating a multi-cluster service that exposes endpoints from a number of Kubernetes clusters
+
+This directory contains the instructions on how to create a multi-cluster service that exposes endpoints on fleet.
+
+The application is using [kuard](https://github.com/kubernetes-up-and-running/kuard) as the service to demonstrate the responding cluster/pod.
+
+## Before you begin
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/).
+* Create a fleet with two members: `aks-member-blue`and `aks-member-green` using [Azure CNI networking](https://review.learn.microsoft.com/en-us/azure/aks/configure-azure-cni).
+* Member clusters reside on the same virtual network, or they reside on different virtual networks that are [peer-linked](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview), i.e. Pods from different clusters can communicate directly with each other using their assigned Pod IP addresses.
```
Comment (circy9) [inline examples/fleet/kuard/README.md]: Shorten this?

Member clusters reside on the same virtual network or [peered virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview) such that pods from different clusters can communicate directly with each other using pod IPs.

File: examples/fleet/kuard/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# Example: Creating a multi-cluster service that exposes endpoints from a number of Kubernetes clusters
+
+This directory contains the instructions on how to create a multi-cluster service that exposes endpoints on fleet.
+
+The application is using [kuard](https://github.com/kubernetes-up-and-running/kuard) as the service to demonstrate the responding cluster/pod.
+
+## Before you begin
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/).
+* Create a fleet with two members: `aks-member-blue`and `aks-member-green` using [Azure CNI networking](https://review.learn.microsoft.com/en-us/azure/aks/configure-azure-cni).
```
Comment (circy9) [inline examples/fleet/kuard/README.md]: `aks-member-blue` and

**Review by circy9 — APPROVED** [review]

**Review by circy9 — APPROVED** [review]

File: examples/fleet/kuard-blue-green/README.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+   ```shell
+   KUBECONFIG=member-green kubectl create namespace net
+
+   KUBECONFIG=member-green kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard-blue-green/kuard-green.yaml
+   ```
+
+### 3. Create service and service export in the member clusters
+
+1. Switch to `aks-member-blue` context to create the kuard service with an internal load balancer and service export:
```
Comment (ryanzhang-oss) [inline examples/fleet/kuard-blue-green/README.md:52]: not internal local balancer anymore, right?

File: examples/fleet/kuard-blue-green/README.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+   kuard   kuard            20.81.3.74    True       26s
+   ```
+3. Access the running service:
+
+   ```shell
+   curl 20.81.3.74:8080
+   ```
+   Or access the http://20.81.3.74:8080 in the browser.
+
+   The request will randomly hit different hostnames with blue or green version.
```
Comment (ryanzhang-oss) [inline examples/fleet/kuard-blue-green/README.md:116]: is this going to be proportional to the number of pods in each deployment? (I know it is :) )

**Review by ryanzhang-oss — APPROVED** [review]

**Pull request merged**